gulp/.eslintrc.json

```diff
@@ -0,0 +1,5 @@
+{
+    "rules": {
+        "no-console": "off"
```

Comment:
@bjacobel I quite like this rule, forcing people to merge cleaner code. Is there a reason we would want ```console.log``` statements in the codebase? I would be fine keeping ```warn``` or ```error``` if it is needed but vote to get rid of debugging statements from the code.